src/SmartFormat.Tests/TestUtils/ConvertibleDecimal.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace SmartFormat.Tests.TestUtils;
+
+class ConvertibleDecimal : IConvertible
+{
+    public decimal value;
```

Comment:
Make code style happy with `private readonly decimal _value;`?

Reply:
Ah yes ill rename it